packages/openneuro-app/src/scripts/dataset/draft-container.tsx

```diff
@@ -194,7 +205,8 @@ const DraftContainer: React.FC<DraftContainerProps> = ({ dataset }) => {
                 editMode={hasEdit}
               >
                 {description?.Authors?.length ? description.Authors : ["N/A"]}
-              </EditDescriptionList>
+              </EditDescriptionList> */
```

Comment:
I think we do want to be able to edit this still. It should create the datacite.yml entries, maybe this can be done in another PR?

Reply:
yeah. another PR is good.